slides.Rmd

```diff
@@ -321,24 +364,21 @@ borders_filtered <- filter(borders,
 ```
 
 
-
 ## Exercise 2:
```

Comment:
We introduce dplyr::between() here as an alternate / cleaner way to answer (after they had done it themselves using operators etc.)

Reply:
Thanks, I didn't know about this function! I've added a Trainers Note to do this.